Tasks/AzureRmWebAppDeploymentV4/operations/KuduServiceUtility.ts

```diff
@@ -78,8 +77,13 @@ export class KuduServiceUtility {
         }
         finally {
             try {
+                let deleteFilePath = '%Home%\\site\\VSTS_PostDeployment_' + uniqueID + '\\delete_log_file' + fileExtension;
+                if(taskParams.isLinuxApp) {
+                    deleteFilePath = '/home/site/VSTS_PostDeployment_' + uniqueID + '/delete_log_file' + fileExtension;
+                }
+                
                 await this._appServiceKuduService.uploadFile(vstsPostDeploymentFolderPath, 'delete_log_file' + fileExtension, path.join(__dirname, '..', 'postDeploymentScript', 'deleteLogFile' + fileExtension));
-                await this.runCommand(vstsPostDeploymentFolderPath, 'delete_log_file' + fileExtension, 0, null);
+                await this.runCommand(vstsPostDeploymentFolderPath, deleteFilePath);
```

Comment:
should we append file extension?

Reply:
deleteFilePath variable has the extension included.